zfsbackup.py

```diff
@@ -80,6 +80,7 @@ def main():
             lf_path = conf.get('lock_file')
         # TODO future: user selectable logging levels
         logging.getLogger().setLevel(logging.INFO)
+        retain_snaps = conf.get('retain_snaps')
```

Comment:
might want to conditionally set this like the other global confs in case it's not defined

Reply:
It'll be None if it's not set, which the clean_dest_snaps function uses in conjunction with the lack of destination retain_snaps to decide not to delete any snaps.